itests/utilities/common_tx_utility.go

```diff
@@ -458,3 +471,26 @@ func BroadcastAndWaitTransaction(suite *f.BaseSuite, tx proto.Transaction, schem
 	errWtGo, errWtScala := suite.Clients.WaitForTransaction(id, timeout)
 	return *NewConsideredTransaction(id, respGo, respScala, errWtGo, errWtScala, errBrdCstGo, errBrdCstScala)
 }
+
+func getItestsDir() (string, error) {
+	filename, err := os.Getwd()
+	if err != nil {
+		return "", err
+	}
+	return filepath.Join(filepath.Dir(filename), "itests"), nil
+}
+
+func ReadScript(scriptDir, fileName string) ([]byte, error) {
+	cutCommentsRegex := regexp.MustCompile(`\s*#.*\n?`)
```

Comment:
Please, move this to a new global variable. Because now regexp needs to be compiled in each function call. (It's not necessary, but nice to have)

Reply:
Done